dashboard/app/controllers/sessions_controller.rb

```diff
@@ -6,6 +6,12 @@ class SessionsController < Devise::SessionsController
   def new
     session[:return_to] ||= params[:return_to]
     @already_hoc_registered = params[:already_hoc_registered]
+    @hide_oauth_sidebar = false
+    if params[:providerNotLinked]
+      flash.now[:alert] = I18n.t 'auth.not_linked', provider: I18n.t("auth.#{params[:providerNotLinked]}")
```

Comment:
Hmm, not sure I'm a fan of our provider space driving i18n strings. Is this a product requirement?

If we do go this route, this PR should carefully document (in `models/user.rb`?) that additional i18n strings need to be created when a new provider is hooked up.